spec/formatterSpec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var formatDebitTransaction = require('../src/formatter').formatDebitTransaction
```

Comment:
I'm not quite sure why each function is imported separately, rather than importing the whole `formatter` module.